### PR TITLE
chore: connection wizard tip-boxes

### PR DESCRIPTION
**Pull request by vasilev-alex**

### CODE REVIEW

File: packages/cubejs-playground/src/pages/ConnectionWizard/ConnectionWizardPage.tsx [hunk 1]
```diff
@@ -110,16 +110,29 @@ export default function ConnectionWizardPage({ history }) {
...
-                  Cube.js will store your credentials into the .env file for
-                  future use.
-                </p>
+                  Cube.js will store your credentials into the <b>.env</b> file
+                  for future use.
+                </Typography.Paragraph>
               )}
+
+              <Typography.Paragraph>
+                For advanced configuration use cube.js configuration file inside
```
Comment (hassankhan) [inline packages/cubejs-playground/src/pages/ConnectionWizard/ConnectionWizardPage.tsx:123]:
```suggestion
                For advanced configuration, use the <code>cube.js</code> configuration file inside
```

File: packages/cubejs-playground/src/pages/ConnectionWizard/ConnectionWizardPage.tsx [hunk 2]
```diff
@@ -110,16 +110,29 @@ export default function ConnectionWizardPage({ history }) {
...
+                mount volume or environment variables.<br />
+                <Typography.Link href="https://cube.dev/docs/connecting-to-the-database" target="_blank">
+                  Learn more about connecting to the databases in the docs.
```
Comment (hassankhan) [inline packages/cubejs-playground/src/pages/ConnectionWizard/ConnectionWizardPage.tsx:126]:
```suggestion
                  Learn more about connecting to databases in the documentation.
```

**Review by hassankhan — COMMENTED** [review]
Minor changes, looks great otherwise 👍

**Comment by vasilev-alex** [comment]
@hassankhan thanks!

**Pull request merged**